docs/outlook/debug-autolaunch.md

```diff
@@ -0,0 +1,136 @@
+- [Configure Visual Studio Code](#configure-visual-studio-code)
+- [Attach VS Code](#attach-vs-code)
+- [Debug](#debug)
+
+You have several options for creating your add-in project. In a couple of stages, the steps vary depending on the option you're using. Where this is the case, if you used the Yeoman generator for Office Add-ins to create your add-in project (for example, by doing the [event-based activation walkthrough](autolaunch.md)), then follow the **yo office** steps, else follow the **Other** steps.
+
+## How to preview debugging for the event-based activation feature
+
+We invite you to try out the debugging capability for the event-based activation feature! Let us know your scenarios and how we can improve by giving us feedback through GitHub (see the **Feedback** section at the end of this page).
+
+To preview this capability for Outlook on Windows, the minimum required build is 16.0.13729.20000. Join the [Office Insider program](https://insider.office.com) for access to Office beta builds.
+
+## Mark your add-in for debugging
+
+1. Set the registry key `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\Wef\Developer\[Add-in ID]\UseDirectDebugger`. `[Add-in ID]` is the **Id** in the add-in manifest.
+
+    **yo office**: In a command line window, navigate to the root of your add-in folder then run the following command.
+
+    ```command&nbsp;line
+    npm start
+    ```
+
+    In addition to building the code and starting the local server, this command should set the `UseDirectDebugger` registry key for this add-in to `1`.
+
+    **Other**: Add the `UseDirectDebugger` registry key under `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\WEF\Developer\[Add-in ID]\`. Replace `[Add-in ID]` with the **Id** from the add-in manifest. Set the registry key to `1`.
+
+    [!include[Developer registry key](../includes/developer-registry-key.md)]
+
+1. Start Outlook desktop (or restart Outlook if it's already open).
+1. Compose a new message or appointment. You should see the following dialog. Do *not* interact with the dialog yet.
+
+    ![Screenshot of Debug Event-based handler dialog](../images/outlook-win-autolaunch-debug-dialog.png)
+
+## Configure Visual Studio Code
+
+### yo office
+
+1. Back in the command line window, open VS Code.
+
+    ```command&nbsp;line
+    code .
+    ```
+
+1. In VS Code, open the file **./.vscode/launch.json** and add the following excerpt to your list of configurations. Save your changes.
+
+    ```json
+    {
+      "name": "Direct Debugging",
+      "type": "node",
+      "request": "attach",
+      "port": 9229,
+      "protocol": "inspector",
+      "timeout": 600000,
+      "trace": true
+    }
+    ```
+
+### Other
+
+1. Create a new folder called **Debugging** (perhaps in your **Desktop** folder).
+1. Open VS Code.
+1. Go to **File** > **Open Folder**, navigate to the folder you just created, then choose **Select Folder**.
+1. Select the **Debug** item in the left menu.
```

Comment:
```suggestion
1. On the Side Bar, select the **Debug** item (CTRL + Shift + D).
```
Note that the Side Bar can be on either side of the screen. I keep mine on the right!

Reply:
Based on the [style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=27401) and [VSC UI](https://code.visualstudio.com/docs/getstarted/userinterface), shouldn't this say:
```suggestion
1. On the Activity Bar, select the **Debug** item (Ctrl+Shift+D).
```